Assets/Shopify/UIToolkit/ShopControllers/MultiProductShopController.cs

```diff
@@ -5,6 +5,7 @@
     using UnityEngine;
 
     public class MultiProductShopController : ShopControllerBase {
+        private bool _isLoadingForFirstTime = true;
```

Comment:
This is a way to track weather or not the product page should display a loading state, on the first load we put up the loading overlay and after that first load completes we can put it away.